test/process_client.py

```diff
@@ -70,66 +76,23 @@ def wait_for_rest(url, max_attempts, sleeping_time, keys_to_check, headers=None)
             raise Exception('Max attempts reached!')
 
 
-def publish_layer(username,
-                  layername,
-                  file_paths=None,
-                  headers=None,
-                  access_rights=None,
-                  title=None,
-                  assert_status=True,
-                  ):
-    title = title or layername
-    headers = headers or {}
-    file_paths = file_paths or ['sample/layman.layer/small_layer.geojson']
-
-    with app.app_context():
-        r_url = url_for('rest_layers.post', username=username)
-
-    for fp in file_paths:
-        assert os.path.isfile(fp)
-    files = []
-    try:
-        files = [('file', (os.path.basename(fp), open(fp, 'rb'))) for fp in file_paths]
-        data = {'name': layername,
-                'title': title,
-                }
-        if access_rights and access_rights.get('read'):
-            data["access_rights.read"] = access_rights['read']
-        if access_rights and access_rights.get('write'):
-            data["access_rights.write"] = access_rights['write']
-        r = requests.post(r_url,
-                          files=files,
-                          data=data,
-                          headers=headers)
-        if assert_status:
-            assert r.status_code == 200, r.text
-
-    finally:
-        for fp in files:
-            fp[1][1].close()
-
-    with app.app_context():
-        url = url_for('rest_layer.get', username=username, layername=layername)
-    if assert_status:
-        wait_for_rest(url, 30, 0.5, layer_keys_to_check, headers=headers)
-    else:
-        return r
-    return layername
-
-
-def patch_layer(username,
-                layername,
-                file_paths=None,
-                headers=None,
-                access_rights=None,
-                title=None,
-                assert_status=True,
-                ):
+def patch_publication(publication_type,
```

Comment:
I am not sure about order (publication_type, workspace, publication_name). It's probably more natural to use order as it is in REST API URL, so (workspace, publication_type, publication_name). Let's discuss it together.

Reply:
The reason for this order is same as in other functions, we want to use with `partial`. For example in src/layman/common/filesystem/util.py::get_publication_dir and many others in the same module.